### PR TITLE
Removes most of the loadout

### PR DESCRIPTION
**Pull request by LordFowl**
This PR removes several of the loadout items in compliance with the new expectations set by Skull
1: Loadout items should not be outlandish
2: Loadout items should not be ugly

- [x] Remove outlandish loadout items

- [x] Remove ugly loadout items

- [x] Remove ugly hairstyles

### CODE REVIEW

**Comment by LordFowl** [comment]
Please don't review/approve WIP PRs.

**Comment by OneOneThreeEight** [comment]
A review is not necessarily permanent.

**Comment by ParadoxSpace** [comment]
Why ponchos?

**Comment by OneOneThreeEight** [comment]
Because he said so. Now stop arguing. Just let it happen.

**Comment by ParadoxSpace** [comment]
No.

**Comment by alsoandanswer** [comment]
Simple fix would be making all poncho wearers automatically valid

Tidying up hairstyles would be nice but it would be a case by case basis - depends on how many people already using them

**Comment by VTCobaltblood** [comment]
Looks quite good so far, and I support the idea. More suggestions for removal:

- Sombreros
- Cowboy hats
- Winter boots, since you've removed the winter coats
- ALL rainbow items (gloves, jumpsuits, caps, shoes)
- Some of the dresses (the blue flames one pops up in my head the quickest)
- "shorts shorts"
- Tacticool turtleneck
- The Vegeta and FF Cloud hairstyles

I will be respriting ponchos for them to look like, well, ponchos in a future PR.

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 1]
```diff
@@ -1,9 +1,3 @@
-/datum/gear/head
```
Comment (Alberyk) [inline code/modules/client/preference_setup/loadout/loadout_head.dm]: I think that removing this datum will affect all the other head ones, due to it being the parent for all others.

**Comment by Conspiir** [comment]
I'm against this mainly for the winter coats. Civ jobs should be allowed to have them, at least. 

In fact, visitors should be wearing whatever they like. They bring it from home to wear.

**Comment by ParadoxSpace** [comment]
On behalf of Lord Jackboot,
Keep poncho. Its been a unathi centric item for years now. 
Keep iac outfit. Its a lore item and it fits the scheme of medical as identifiable.
These are a preliminary. Full coming later.

**Comment by LordFowl** [comment]
The poncho is not an Unathi item. It would be like keeping cloaks in because Vaurca sometimes can wear cloaks.
The IAC outfit was not removed. If Jackboot wants to provide feedback, he is free to actually read the PR first.

Ultimately as these removals are not affecting xeno species, I will take what Jackboot has to suggest as just that - a suggestion. If he wants Unathi to wear ponchos I encourage him to justify and code it, or inevitably find someone else to.

**Comment by Tomiix** [comment]
Heya! Just wanted to provide feedback for some unintended consequences of this.

Unathi players who play Cook/Robotics are hurt by the removal of winter coats as a functional change, due to the fact that they kept you warm when entering their respective freezers.

If the issue is that they are unappealing to the eye, a redesign or a equally as functional option would be helpful.

**Comment by VTCobaltblood** [comment]
As a compromise, I will be making ponchos warm in my resprite. The issue, or at least my personal opinion on the issue with winter coats is that they're not appropriate anywhere _but_ the freezer.

**Comment by LordFowl** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 2]
```diff
@@ -123,6 +122,10 @@
 	path = /obj/item/clothing/accessory/storage/pouches
 	cost = 2
 
+/datum/gear/accessory/locket
```
Comment (Alberyk) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm]: This is already present in the loadout.

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 3]
```diff
@@ -195,15 +187,6 @@
 	..()
 	gear_tweaks = list(gear_tweak_free_color_choice)
 
-/datum/gear/head/loose_beanie
```
Comment (Alberyk) [inline code/modules/client/preference_setup/loadout/loadout_head.dm]: Not sure what is wrong with this item in particular?

File: code/modules/mob/abstract/new_player/sprite_accessories.dm [hunk 4]
```diff
@@ -416,43 +384,17 @@
 	balding
 		name = "Balding Hair"
 		icon_state = "hair_e"
-		gender = MALE // turnoff!
```
Comment (Alberyk) [inline code/modules/mob/abstract/new_player/sprite_accessories.dm]: Is this on purpose?

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm (hunk 2 above)
Comment (LordFowl) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm]: It was actually duplicated, I deleted one lf the duplicates. Not sure why Git decided to represent the diff like this

File: code/modules/client/preference_setup/loadout/loadout_head.dm (hunk 3 above)
Comment (LordFowl) [reply code/modules/client/preference_setup/loadout/loadout_head.dm]: Poorly sprited.

File: code/modules/mob/abstract/new_player/sprite_accessories.dm (hunk 4 above)
Comment (LordFowl) [reply code/modules/mob/abstract/new_player/sprite_accessories.dm]: The to-do comment says to turn off the bit-flag, but no one got around to it.

Comment (ParadoxSpace) [reply code/modules/mob/abstract/new_player/sprite_accessories.dm]: The to-do comment says that the hairstyle is a turnoff.

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 5]
```diff
@@ -123,10 +119,6 @@
 	hats["hat, black fedora"] = /obj/item/clothing/head/fedora
 	hats["hat, brown fedora"] = /obj/item/clothing/head/fedora/brown
 	hats["hat, grey fedora"] = /obj/item/clothing/head/fedora/grey
-	hats["hat, beaver"] = /obj/item/clothing/head/beaverhat
-	hats["hat, cowboy"] = /obj/item/clothing/head/cowboy
```
Comment (ParadoxSpace) [inline code/modules/client/preference_setup/loadout/loadout_head.dm]: for what purpose

File: code/modules/client/preference_setup/loadout/loadout_head.dm [hunk 6]
```diff
@@ -123,10 +119,6 @@
...
-	hats["hat, wide-brimmed cowboy"] = /obj/item/clothing/head/cowboy/wide
```
Comment (ParadoxSpace) [inline code/modules/client/preference_setup/loadout/loadout_head.dm]: also here. why?

File: code/modules/client/preference_setup/loadout/loadout_shoes.dm [hunk 7]
```diff
@@ -60,18 +55,6 @@
 	shoes["dress flats, purple"] = /obj/item/clothing/shoes/flats/purple
 	gear_tweaks += new/datum/gear_tweak/path(shoes)
 
-/datum/gear/shoes/cowboy
```
Comment (ParadoxSpace) [inline code/modules/client/preference_setup/loadout/loadout_shoes.dm]: for what purpose

File: code/modules/mob/abstract/new_player/sprite_accessories.dm [hunk 8]
```diff
@@ -253,10 +239,6 @@
 		name = "Devil Lock"
 		icon_state = "hair_devilock"
 
-	dreadlocks
```
Comment (ParadoxSpace) [inline code/modules/mob/abstract/new_player/sprite_accessories.dm]: why these

Comment (VTCobaltblood) [reply code/modules/mob/abstract/new_player/sprite_accessories.dm]: Ugly.

File: code/modules/mob/abstract/new_player/sprite_accessories.dm [hunk 9]
```diff
@@ -509,6 +441,10 @@
 		name = "Modern"
 		icon_state = "hair_modern"
 
+	father
```
Comment (VTCobaltblood) [inline code/modules/mob/abstract/new_player/sprite_accessories.dm]: sneaky

File: code/modules/mob/abstract/new_player/sprite_accessories.dm [hunk 10]
```diff
@@ -676,14 +599,6 @@
 		name = "Naomi Mohawk"
 		icon_state = "hair_mohawkshavednaomi"
 
-	leftsidecut
```
Comment (VTCobaltblood) [inline code/modules/mob/abstract/new_player/sprite_accessories.dm]: Same here. Dreg culture.

File: code/modules/mob/abstract/new_player/sprite_accessories.dm [hunk 11]
```diff
@@ -652,14 +583,6 @@
 		name = "Half-Shaved"
 		icon_state = "hair_halfshaved"
 
-	halfshavedemo
```
Comment (VTCobaltblood) [inline code/modules/mob/abstract/new_player/sprite_accessories.dm]: Why? It's dreg culture.

File: code/modules/mob/abstract/new_player/sprite_accessories.dm (hunk 8 above)
Comment (ParadoxSpace) [reply code/modules/mob/abstract/new_player/sprite_accessories.dm]: explain

**Pull request closed without merging**